kotlin-analysis-api/src/test/kotlin/com/google/devtools/ksp/impl/test/KSPAATest.kt

```diff
@@ -321,6 +321,13 @@ class KSPAATest : AbstractKSPAATest() {
         runTest("../compiler-plugin/testData/api/javaWildcards2.kt")
     }
 
+    @Disabled
+    @TestMetadata("lateinitProperties.kt")
+    @Test
+    fun testLateinitProperties() {
+        runTest("testData/api/lateinitProperties.kt")
```

Comment:
should be `"../compiler-plugin/testData/api/lateinitProperties.kt"`

Reply:
Done